responsibleai/responsibleai/feature_metadata.py

```diff
@@ -102,6 +115,29 @@ def __eq__(self, other_feature_metadata) -> bool:
             self.time_series_id_features == \
             other_feature_metadata.time_series_id_features
 
+    @staticmethod
+    def _validate_column_list(
+            column_purpose: str,
+            column_names: Any):
+        """Ensure the provided column list is valid list of strings.
```

Comment:
```suggestion
        """Ensure the provided column list is a list of strings.
```
We're not checking for validity of the strings themselves (i.e., are they actually columns of the dataset) so in essence just checking whether it's a list of strings.